From b1f094af008ed18eee45c6733d6e607a1f493bce Mon Sep 17 00:00:00 2001
From: Thierry FOURNIER <tfournier@arpalert.org>
Date: Fri, 25 Dec 2015 02:05:25 +0100
Subject: [PATCH 39/67] DOC: lua: fix somme errors

This patch fix some errors in the class TXN doc.

Should be backported in 1.6
(cherry picked from commit 834421c2d0226a567c2692ba52df7b14a6bc4c1d)
---
 doc/lua-api/index.rst | 7 -------
 1 file changed, 7 deletions(-)

diff --git a/doc/lua-api/index.rst b/doc/lua-api/index.rst
index 7eef9e4..45f2b53 100644
--- a/doc/lua-api/index.rst
+++ b/doc/lua-api/index.rst
@@ -1094,13 +1094,6 @@ TXN class
   :param class_txn txn: The class txn object containing the data.
   :param string var: The variable name according with the HAProxy variable syntax.
 
-.. js:function:: TXN.get_headers(txn)
-
-  This function returns an array of headers.
-
-  :param class_txn txn: The class txn object containing the data.
-  :returns: an array of headers.
-
 .. js:function:: TXN.done(txn)
 
   This function terminates processing of the transaction and the associated
-- 
2.4.10

